core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/TimePartitionedFileSetSource.java

```diff
@@ -99,12 +99,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
 
   @Override
   public final void prepareRun(BatchSourceContext context) {
-    Map<String, String> runtimeArgs = context.getRuntimeArguments();
     long runtime = context.getLogicalStartTime();
-    if (runtimeArgs.containsKey("runtime")) {
-      runtime = Long.parseLong(runtimeArgs.get("runtime"));
```

Comment:
yeah this was a workaround before, but now we have the ability to set 'logical.start.time' as a runtime argument to override the normal logical start time.